docs/vickywane-ambianic-stripe-swagger-docs-0.1-resolved.yaml

```diff
@@ -0,0 +1,51 @@
+---
+openapi: 3.0.1
+info:
+  title: defaultTitle
+  description: defaultDescription
```

Comment:
@vickywane please describe the purpose of this API. It will be visible in the published docs.